pytest_docker_tools/utils.py

```diff
@@ -29,3 +31,11 @@ def tests_inside_container():
     """ Returns True if tests are running inside a Linux container """
 
     return os.path.isfile("/.dockerenv") or os.path.isfile("/run/.containerenv")
+
+
+def is_reusable_container(container):
```

Comment:
This is much better 😄. What do you think about some tweaks to make it fit on one line?

```python
    return container.attrs["Config]["Labels"].get(LABEL_REUSABLE_CONTAINER) == "True"
```

That comes in at 86 characters which is less than the Black default wrap point.

Reply:
Good Point! Adjustment done.